src/lib/crypto/kimchi_backend/common/tests/test_lookup_table_constraint_kind.ml

```diff
@@ -4,14 +4,35 @@ open Kimchi_backend_common.Plonk_constraint_system.Plonk_constraint
 let test_finalize_and_get_gates_with_lookup_tables () =
   let module Tick = Kimchi_backend.Pasta.Vesta_based_plonk in
   let cs = Tick.R1CS_constraint_system.create () in
+  let xor_table =
+    [| [| Tick.Field.zero; Tick.Field.zero; Tick.Field.zero |]
+     ; [| Tick.Field.zero; Tick.Field.one; Tick.Field.one |]
+     ; [| Tick.Field.one; Tick.Field.zero; Tick.Field.one |]
+     ; [| Tick.Field.one; Tick.Field.one; Tick.Field.one |]
+    |]
+  in
+  let and_table =
```

Comment:
Do we have an AND table now?

Reply:
It is an example, but we can have any lookup table.